pkg/obfuscator/target_obfuscator.go

```diff
@@ -21,7 +21,7 @@ func (t *targetObfuscator) Contents(s string) string {
 	return s
 }
 
-func (t *targetObfuscator) Report() map[string]string {
+func (t *targetObfuscator) Report() ReplacementReport {
```

Comment:
while you're already touching this piece, can you embed the obfuscator? then we can also kill that reporting method.